src/app/(authenticated)/membro/[id]/components/InformationMember.tsx

```diff
@@ -0,0 +1,134 @@
+          <div>
+            <h1 className="mb-1 text-xl font-semibold">{user?.name}</h1>
+            <Badge variant="primary">{user?.role}</Badge>
+            <div className="mt-4 flex items-center gap-2">
+              {" "}
+              <Icon className="fill-neutral-100" size={20} />
+              <p className="text-sm">{user?.class}</p>
+            </div>
+          </div>
+        </div>
+        <div>
+          <div className="flex h-6 items-center justify-center gap-4">
+            <div className="flex gap-2">
+              <Discord /> <span className="text-sm">.treffy</span>
+            </div>
+            <div className="flex h-3 w-3 items-center justify-center rounded border border-neutral-700 bg-neutral-800 p-3">
+              <button>
+                <Copy size={12} />
+              </button>
+            </div>
+          </div>
+          <div className="mt-3">
+            <p className="text-right text-sm text-neutral-400">convidado por</p>
+            <h1 className="mt-2 flex justify-end gap-2 font-semibold">
+              <Avatar
+                fallbackText={user?.name}
+                key={user?.image}
+                size={24}
+                src={user?.image}
+              />{" "}
+              Snow
+            </h1>
+          </div>
+        </div>
+      </div>
+      {/*fechamento do header*/}
+      <div className="relative z-10 mt-14 space-y-6">
+        <div>
+          <div className="grid grid-cols-2 gap-8">
+            <div>
+              <h2 className="font-semibold">Nome de Exibição</h2>
+              <p className="text-xs text-neutral-400">
+                Este nome será usado em rankings, organização de eventos, lances
+                e logs. Use o mesmo do jogo.
+              </p>
+            </div>
+            <Input
+              className="h-9 border border-neutral-700 bg-neutral-900"
+              value={nameInput}
+              onChange={(e) => setNameInput(e.target.value)}
+            />
+          </div>
+        </div>
+        <div>
+          <div className="grid grid-cols-2 gap-8">
+            <div>
+              <h2 className="font-semibold">Imagem do Perfil</h2>
+              <p className="text-xs text-neutral-400">
+                Você pode atualizar essa imagem com a sua atual imagem do
+                discord ou pode fazer upload da imagem do seu personagem do
+                jogo.
+              </p>
+            </div>
+            <div className="space-y-4">
+              <Button
+                className="h-29 w-full border border-neutral-700 bg-neutral-800 py-1"
+                variant="outline"
+              >
+                {" "}
+                <Discord /> Usar imagem do Discord
+              </Button>
+              <Button
+                className="h-29 w-full border border-neutral-700 bg-neutral-800 py-1"
+                variant="outline"
+              >
+                {" "}
+                <Camera size={20} /> Fazer um upload
```

Comment:
O tamanho deste icone é 16x16.
![image](https://github.com/user-attachments/assets/c2c6c17e-ef67-4583-8dda-20f31ca5b24e)
Se eu não me engano, size={20} é o default to lucide